spec/features/idv/flow_spec.rb

```diff
@@ -10,7 +10,9 @@
     end
 
     scenario 'decline to verify identity' do
+      click_idv_begin
```

Comment:
Did we remove the "Cancel" link on the `/verify` endpoint on purpose? Are we no longer allowing people to cancel at this point? The only way back seems to be the browser button. Even the login.gov logo is not clickable on that page. Was all of this intentional?

Reply:
@gemfarmer and I talked about removing it from this page as the user hadn't actually started the verification process at this point. But I appreciate your comment -- now there is no way for users to see the information in that cancel modal, or ability to go back to their login.gov account. @gemfarmer let's bring back the cancel link here?

Reply:
@mkhandekar @monfresh I can bring back the link!